client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/utils.ts

```diff
@@ -111,24 +111,34 @@ export const allocateQuantities =
       });
     }
 
-    if (status === InvoiceNodeStatus.New && toAllocate > 0) {
+    if (status === InvoiceNodeStatus.New) {
       const placeholderIdx = newDraftOutboundLines.findIndex(
         ({ type }) => type === InvoiceLineNodeType.UnallocatedStock
       );
       const placeholder = newDraftOutboundLines[placeholderIdx];
+      const oldPlaceholder = draftOutboundLines[placeholderIdx];
+      const placeholderRemoved =
+        (oldPlaceholder?.numberOfPacks ?? 0 > 0) &&
```

Comment:
is 0 > 0 needed? 🤔

Reply:
No, it's not. I've changed to `oldPlaceholder?.numberOfPacks` but also added a comment, as I don't think it's immediately obvious